src/components/views/voip/CallView/CallViewHeader.tsx

```diff
@@ -0,0 +1,118 @@
+import { CallType } from 'matrix-js-sdk/src/webrtc/call';
+import { Room } from 'matrix-js-sdk/src/models/room';
+import React from 'react';
+import { _t } from '../../../../languageHandler';
+import RoomAvatar from '../../avatars/RoomAvatar';
+import AccessibleButton from '../../elements/AccessibleButton';
+import dis from '../../../../dispatcher/dispatcher';
+import WidgetAvatar from '../../avatars/WidgetAvatar';
+import { IApp } from '../../../../stores/WidgetStore';
+import { isUndefined } from 'lodash';
```

Comment:
All the absolute imports tend to be added first

Reply:
Shall we add some sort imports eslint rule?